tactics/tactics.ml

```diff
@@ -375,7 +375,7 @@ let convert_gen pb x y =
     | None ->
       let info = Exninfo.reify () in
       Tacticals.tclFAIL ~info (str "Not convertible")
-    | exception e ->
+    | exception e when CErrors.noncritical e || CErrors.is_anomaly e ->
```

Comment:
Why is this one different?

Reply:
`is_anomaly` means exception without a printer
I think `noncritical || is_anomaly` is equivalent to `noncritical`

Reply:
That's not how I read it:
```
let is_anomaly = function
| Anomaly _ -> true
| exn -> not (is_handled exn)
```
So it corresponds to either (1) `Anomaly` or (2) something that has no printer.
On the other hand, `noncritical` does not include `Anomaly`:
```
let noncritical = function
  | Sys.Break | Out_of_memory | Stack_overflow
  | Assert_failure _ | Match_failure _ | Anomaly _
  | Control.Timeout -> false
  | Invalid_argument "equal: functional value" -> false
  | _ -> true
```

Reply:
right I got confused

I think catching is_anomaly here makes no sense

Just do noncritical and if something fails we can look at it

Reply:
Done.